workbench/src/renderer/components/PluginModal/index.jsx

```diff
@@ -29,36 +34,88 @@ export default function PluginModal(props) {
       if (addPluginErr) {
         setErr(addPluginErr);
       } else {
-        setShowAddPluginModal(false);
+        setShowPluginModal(false);
       }
     });
   };
-  const handleChange = (event) => {
-    setURL(event.currentTarget.value);
+
+  const removePlugin = () => {
+    setLoading(true);
+    ipcRenderer.invoke(ipcMainChannels.REMOVE_PLUGIN, pluginToRemove).then(() => {
+      updateInvestList();
+      setLoading(false);
+      setShowPluginModal(false);
+    });
   };
 
+  useEffect(() => {
+    ipcRenderer.invoke(ipcMainChannels.GET_SETTING, 'plugins').then(
+      (data) => {
+        setPlugins(data);
+        setPluginToRemove(Object.keys(data)[0]);
+      }
+    );
+  }, [loading]);
+
   const { t } = useTranslation();
 
   let modalBody = (
     <Modal.Body>
       <Form>
         <Form.Group className="mb-3">
-          <Form.Label htmlFor="url">Git URL</Form.Label>
+          <Form.Label htmlFor="url">Add a plugin</Form.Label>
           <Form.Control
             id="url"
             name="url"
             type="text"
             placeholder={t('Enter Git URL')}
-            onChange={handleChange}
+            onChange={(event) => setURL(event.currentTarget.value)}
           />
+          <Form.Text className="text-muted">
+            {t('This may take several minutes')}
+          </Form.Text>
+          <Button
+            name="submit"
```

Comment:
Can you remove the `name` attributes from the `button`s? Since we're not submitting raw form data to a server (instead, we're using an explicit `onClick` handler to pre-process data on the front end), there's no need for the `button`s to have `name`s.